README.md

```diff
@@ -5,19 +5,52 @@ A pure Lua Neovim plugin for automatically rendering PlantUML diagrams in a loca
 
 ## Installation
 
-Lazy:
+### Lazy (minimal setup)
+```lua
+{ "charlesnicholson/plantuml.nvim" }
 ```
-  { "charlesnicholson/plantuml.nvim" }
+
+### Lazy (with configuration)
+```lua
+{
+  "charlesnicholson/plantuml.nvim",
+  opts = {
+    auto_start = true,
+    http_port = 8764,
+    host = "127.0.0.1",
```

Comment:
Remove this as well if, via the other comment, `host` gets removed from the plugin config.